preview/Win7Msix/Win7MSIXInstaller/InstallUI.cpp

```diff
@@ -137,13 +137,20 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
     {
         switch (::GetDlgCtrlID((HWND)lParam))
         {
-        case IDC_LAUNCHCHECKBOX:
-        {
-            HBRUSH hbr = (HBRUSH)DefWindowProc(hWnd, message, wParam, lParam);
-            ::DeleteObject(hbr);
-            SetBkMode((HDC)wParam, TRANSPARENT);
-            return (LRESULT)::GetStockObject(NULL_BRUSH);
-        }
+            case IDC_LAUNCHCHECKBOX:
+            {
+                HBRUSH hbr = (HBRUSH)DefWindowProc(hWnd, message, wParam, lParam);
+                ::DeleteObject(hbr);
+                SetBkMode((HDC)wParam, TRANSPARENT);
+                return (LRESULT)::GetStockObject(NULL_BRUSH);
+            }
+            case IDC_STATICPERCENTCONTROL:
+            {
+                HBRUSH hbr = (HBRUSH)DefWindowProc(hWnd, message, wParam, lParam);
+                ::DeleteObject(hbr);
+                SetBkMode((HDC)wParam, TRANSPARENT);
+                return (LRESULT)::GetStockObject(NULL_BRUSH);

```

Comment:
is this expected to differ later from IDC_LAUNCHCHECKBOX or would they always be the same?

Reply:
It is just the background color and formatting. Should be the same. Would it be better to remove the case and keep it common for all?

Reply:
if there's some controls that should, and others that shouldn't then you could just have the first case __fallthrough; if all controls should, then you don't need the switch. 

---
In reply to: [279122987](https://github.com/Microsoft/msix-packaging/pull/101#discussion_r279122987) [](ancestors = 279122987)

Reply:
Removed switch case. Ran the installer and verified changes.